chapters/data-analysis.tex

```diff
@@ -1,182 +1,237 @@
+De-identification,\sidenote{\url{https://dimewiki.worldbank.org/wiki/De-identification}}
+at this stage, means stripping the data set of direct identifiers.\sidenote{\url{ https://www.povertyactionlab.org/sites/default/files/resources/J-PAL-guide-to-deidentifying-data.pdf}}
+To be able to do so, you will need to go through your data set and
+find all the variables that contain identifying information.
+Flagging all potentially identifying variables in the questionnaire design stage
+simplifies the initial de-identification process.
+If you haven't done that, that are a few tools that can help you with it.
+JPAL's \texttt{PII scan}, as indicated by its name,
+scans variable names and labels for common string patterns associated with identifying information.\sidenote{
+	\url{https://github.com/J-PAL/PII-Scan}}
+The World Bank's \texttt{sdcMicro}
+lists variables that uniquely identify observations,
+as well as allowing for more sophisticated disclosure risk calculations.\sidenote{
+	\url{http://sdctools.github.io/sdcMicro/articles/sdcMicro.html}}
+The \texttt{iefieldkit} command \texttt{iecodebook}
+lists all variables in a data set and exports an Excel sheet
+where you can easily select which variables to keep or drop.\sidenote{
+	\url{https://dimewiki.worldbank.org/wiki/Iecodebook}}
+
+Once you have a list of variables that contain PII,
+assess them against the analysis plan and ask:
+will this variable be needed for analysis?
+If not, the variable should be dropped.
+Don't be afraid to drop too many variables the first time,
+as you can always go back and remove variables from the list of variables to be dropped,
+but you can not go back in time and drop a PII variable that was leaked
+because it was incorrectly kept.
+Examples include respondent names, enumerator names, interview date, respondent phone number.
+If the variable is needed for analysis, ask:
+can I encode or otherwise construct a variable to use for the analysis that masks the PII,
+and drop the original variable?
+This is typically the case for most identifying information.
+Examples include geocoordinates
+(after constructing measures of distance or area,
+drop the specific location),
+and names for social network analysis (can be encoded to secret and unique IDs).
+If PII variables are strictly required for the analysis itself,
+it will be necessary to keep at least a subset of the data encrypted through the data analysis process.
+If the answer is yes to either of these questions,
+all you need to do is write a script to drop the variables that are not required for analysis,
+encode or otherwise mask those that are required,
+and save a working version of the data.
+
+The resulting de-identified data will be the underlying source for all cleaned and constructed data.
+This is the data set that you will interact with directly during the remaining tasks described in this chapter.
+Because identifying information is typically only used during data collection,
```

Comment:
```suggestion
Because identifying information is typically only used during data collection
```

I think this sentence reads better without that comma

Reply:
I think I'd leave the comma and change the following line:

"Because identifying information is typically only used during data collection,
when teams need to find and confirm the identity of interviewees,"

Reply:
That is better. Previously it was unclear that the mid-clause was a continuation of the first clause and not a clause by its own, but the word _when_ makes that clearer

Reply:
addressed in c9c9a08